.github/workflows/linkchecker.yaml

```diff
@@ -0,0 +1,26 @@
+name: Check links
+
+on:
+  schedule:
+    - cron: "10 7 * * *"
+
+jobs:
+  run-cypress:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v2
+
+      - name: Build image
+        run: DOCKER_BUILDKIT=1 docker build --tag ubuntu-com .
+
+      - name: Run linkchecker against the server
+        run: |
+          docker run -d -p 80:80 ubuntu-com
+          sleep 5
+          linkchecker --threads 2 --ignore-url /usn --ignore-url /search --ignore-url ".*(?:png|jpg|jpeg|gif|bmp|svg|js)$" --ignore-url /q_auto --ignore-url /fl_sanitize --ignore-url /w_* --ignore-url /h_* --ignore-url /blog --ignore-url /server/docs --no-warnings http://127.0.0.1
```

Comment:
I think we should add ` --check-extern  ` parameter to check also sites outside that we point to.

Reply:
let's experiment with this later. I'm worried it will always fail with that parameter and spam us all with emails every morning.

Reply:
ok, if we create an issue for it I am fine with it.